ydb/docs/en/core/integrations/orm/linq2db.md

```diff
@@ -0,0 +1,503 @@
+| `byte[]`                      | `BinaryJson`                              | `JsonDocument`     | Binary JSON.                                                          |
+| `DateOnly` / `DateTime`       | `Date`                                    | `Date32`           | **New type.** Wider date range. Use `DbType = "Date32"`.              |
+| `DateTime`                    | `DateTime`                                | `Datetime64`       | **New type.** Second precision, wider range. `DbType = "Datetime64"`. |
+| `DateTime` / `DateTimeOffset` | `DateTime2`                               | `Timestamp64`      | **New type.** Microseconds, wider range. `DbType = "Timestamp64"`.    |
+| `TimeSpan`                    | `Interval`                                | `Interval64`       | **New type.** Wider interval range. `DbType = "Interval64"`.          |
+
+> You can set exact `Precision`/`Scale` with attributes:  
+    >   `[Column(DataType = DataType.Decimal, Precision = 22, Scale = 9)]`  
+    >   or globally via `YdbOptions(UseParametrizedDecimal: true)`.
+
+> Time zone types (`TzDate`/`TzDatetime`/`TzTimestamp`) are **not used as column types**. When creating tables they will be reduced to `Date`/`Datetime`/`Timestamp`. They are allowed in expressions/literals.
+
+> By default (when DbType is not specified on a column) the provider uses the legacy YDB temporal types: Date, Datetime, Timestamp, Interval.
+To opt in to the new temporal types per column, set DbType on that column, e.g. `[Column(DbType = "Date32")]`. Both families can coexist in the same table.
+---
+
+### Schema generation from attributes
+
+Describe an entity using Linq To DB attributes; the provider will create a table and indexes.
+
+{% list tabs group=lang %}
+
+- C#
+
+  ```csharp
+  using LinqToDB.Mapping;
+
+  [Table(Name = "Groups", IsColumnAttributeRequired = false)]
+  [Index("GroupName", Name = "group_name_index")]
+  public class Group
+  {
+      [PrimaryKey, Column("GroupId")]
+      public int Id { get; set; }
+
+      [Column("GroupName")]
+      public string? Name { get; set; }
+  }
+  ```
+
+{% endlist %}
+
+**Generated DDL ({{ ydb-short-name }})**:
+
+```yql
+CREATE TABLE Groups (
+    GroupId Int32 NOT NULL,
+    GroupName Utf8,
+    PRIMARY KEY (GroupId)
+);
+
+ALTER TABLE Groups
+  ADD INDEX group_name_index GLOBAL
+       ON (GroupName);
+```
+
+### Schema evolution (adding a column)
+
+Add the `Department` column to the `Group` entity:
+
+{% list tabs group=lang %}
+
+- C#
+
+  ```csharp
+  [Column] public string? Department { get; set; }
+  ```
+
+{% endlist %}
+
+**Schema change (DDL):**
+
+```yql
+ALTER TABLE Groups
+   ADD COLUMN Department Utf8;
+```
+
+> Linq To DB does not manage migrations — use Liquibase/Flyway/your tool of choice. Methods like `CreateTable<T>()`/`DropTable<T>()` are convenient for tests and local development.
```

Comment:
Adding a column is a specific kind of migration. Why do we give an example of a migration and then say that they are not supported?

Add the command that would execute this generated query.